src/PackageCompiler.jl

```diff
@@ -378,7 +378,7 @@ function create_sysimg_object_file(object_file::String,
     write(outputo_file, julia_code)
     # Read the input via stdin to avoid hitting the maximum command line limit
 
-    cmd = `$(get_julia_cmd()) --cpu-target=$cpu_target -O3 $sysimage_build_args
+    cmd = `JULIA_DEBUG=loading $(get_julia_cmd()) --cpu-target=$cpu_target -O3 $sysimage_build_args
```

Comment:
Does this work? Don't you need to use `withenv` or something?

Reply:
No it doesn't work. I pushed a fix